src/control/xml/DoubleArrayAttribute.cpp

```diff
@@ -1,31 +1,29 @@
 #include "DoubleArrayAttribute.h"
 #include "Util.h"
 
-DoubleArrayAttribute::DoubleArrayAttribute(const char* name, double* values, int count) : XMLAttribute(name)
+DoubleArrayAttribute::DoubleArrayAttribute(const char* name, std::vector<double>&& values)
```

Comment:
remove the rvalue reference and take the vector per value, since the current version forces to use std::move on the calling side or an explicit copy.

Reply:
I don't see the problem with the move constructor. Anyways, the calling side has been made more concise.